### PR TITLE
Request db file along with log files

### PR DESCRIPTION
**Pull request by arshdeeptinna**
Updated node operator manual view logs section to request node operators to attach log and db files with Github issue
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: doc/node-operator.md [hunk 1]
```diff
@@ -299,7 +299,16 @@ To find all log files:
...
 
 You may have to follow the above process multiple times to debug an issue. 
-If you open up a GitHub issue, please include your findings and [attach all relevant log files](https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests).
+
+If you open up a GitHub issue, please include your findings and [attach all relevant log and db files](https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests).
+
+To find the db file:
+1. `cd` to directory that contains qsp-protocol-node code
+1. `cd resources`
+1. Find the latest db file using `ls -Art *.db | tail -n 1`
```
Comment (martinetlee) [inline doc/node-operator.md]: Do we only need the latest db file or just ask them to submit all db file?

File: doc/node-operator.md [hunk 2]
```diff
@@ -299,7 +299,16 @@ To find all log files:
 1. check the file for relevant error messages 
 
 You may have to follow the above process multiple times to debug an issue. 
-If you open up a GitHub issue, please include your findings and [attach all relevant log files](https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests).
+
+If you open up a GitHub issue, please include your findings and [attach all relevant log and db files](https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests).
+
+To find the db file:
+1. `cd` to directory that contains qsp-protocol-node code
```
Comment (martinetlee) [inline doc/node-operator.md]: `directory` -> `the directory`?

File: doc/node-operator.md (hunk 1 above)
Comment (arshdeeptinna) [reply doc/node-operator.md]: I think DB file don't change very often except for version upgrades for nodes or contracts but @alex-murashkin can answer better

Comment (alex-murashkin) [reply doc/node-operator.md]: Yeah I think there is only one db file. I am not sure why we actually need the `*.db`

Comment (arshdeeptinna) [reply doc/node-operator.md]: fixed

File: doc/node-operator.md [hunk 3]
```diff
@@ -299,7 +299,16 @@ To find all log files:
...
-If you open up a GitHub issue, please include your findings and [attach all relevant log files](https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests).
+
+If you open up a GitHub issue, please include your findings and [attach all relevant log and db files](https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests).
+
+To find the db file:
+1. `cd` to the directory that contains qsp-protocol-node code
+1. `cd resources`
+1. Find the latest db file using `ls event_database.db`
+
+**NOTE**:Please zip the files before attaching to a Github issue
```
Comment (alex-murashkin) [inline doc/node-operator.md]: `GitHub`

**Comment by alex-murashkin** [comment]
Please include meaningful descriptions for the PRs.

**Pull request merged**